### PR TITLE
ADD:SSV-Prysm

### PR DESCRIPTION
**Pull request by gbayasgalan**

### CODE REVIEW

**Comment by stefa2k** [comment]
Please approve https://github.com/stereum-dev/ethereum-node/pull/63 first to get the existing ssv tests successful again. Then we review this one and adapt it if needed.

**Review by stefa2k — CHANGES_REQUESTED** [review]

File: controls/roles/manage-service/molecule/ssv-prysm/molecule.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      ip_range: 10.10.0.0/24
+      ip: 10.10.0.105
+  - name: "manage-service--ssv-prysm--centos-stream-8"
+    hostname: "centos"
+    server_type: cpx31
+    image: centos-stream-8
+    network:
+      name: eth2-prater
+      ip_range: 10.10.0.0/24
+      ip: 10.10.0.106
```
[the remark this answers is not in the document]
Comment (gbayasgalan) [reply controls/roles/manage-service/molecule/ssv-prysm/molecule.yml:22]: 10.10.0.116 is choosen

File: controls/roles/manage-service/molecule/ssv-prysm/prepare.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        include_role:
+          name: "manage-service"
+      vars:
+        stereum_args:
+          manage_service:
+            save: true
+            state: started
+            configuration:
+              id: "{{ beacon_service }}"
+              image: gcr.io/prysmaticlabs/prysm/beacon-chain:v2.0.5
```
[the remark this answers is not in the document]
Comment (gbayasgalan) [reply controls/roles/manage-service/molecule/ssv-prysm/prepare.yml:55]: done!

File: controls/roles/manage-service/molecule/ssv-prysm/verify.yml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    command: "docker logs --tail=150 stereum-995c224c-82e9-11ec-9115-2349bc4566fa"
+    register: beacon
+    until: beacon.stderr is not search('estimated time remaining')
+    retries: 360
+    delay: 10
+  - name: Blox-SSV
+    command: "docker logs --tail=250 stereum-99e8942a-82e9-11ec-9f76-cbc103131365"
+    register: blox_ssv
+    until: 
+      - blox_ssv.stderr.find("beacon node is not healthy") == -1
```
[the remark this answers is not in the document]
Comment (gbayasgalan) [reply controls/roles/manage-service/molecule/ssv-prysm/verify.yml:44]: done!

File: controls/roles/manage-service/molecule/ssv-prysm/verify.yml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+      that:
+        - ssv_config_file.stat.exists
+  - name: Waiting for the services to start properly
+    pause:
+      minutes: 10
+  # prysm beacon & ssv logs
+  - name: prysm beacon node
+    command: "docker logs --tail=150 stereum-995c224c-82e9-11ec-9115-2349bc4566fa"
+    register: beacon
+    until: beacon.stderr is not search('estimated time remaining')
```
[the remark this answers is not in the document]
Comment (gbayasgalan) [reply controls/roles/manage-service/molecule/ssv-prysm/verify.yml:37]: done!

**Review by stefa2k — APPROVED** [review]

**Pull request merged**